api/src/Doctrine/Filter/ContentNodePeriodFilter.php

```diff
@@ -16,6 +17,8 @@
 use Symfony\Component\Serializer\NameConverter\NameConverterInterface;
 
 final class ContentNodePeriodFilter extends AbstractFilter {
+    use FiltersByCampCollaboration;
```

Comment:
We only use this to have access to `queryBuilderAddParameters`.
Should this function go into a separate class (e.g. `QueryBuilderHelper`)?

Reply:
[c0ce12d](https://github.com/ecamp/ecamp3/pull/4935/commits/c0ce12d19527f431267464ce88957ae6f770e940)